task.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+# изменяем текущую директорию на task
+rm -rf ./task
+
+mkdir -p ./task/dir1 ./task/dir2 ./task/dir3/dir4
+cd ./task
```

Comment:
Можно не указывать `./`
```suggestion
cd task
```